iron-list.html

```diff
@@ -761,6 +770,20 @@
       this.unlisten(this, 'iron-resize', '_resizeHandler');
     },
 
+    _itemSizeFn: function(node, is, list) {
+      if (is === ITEM_WIDTH) {
+        return list.grid ? node.getBoundingClientRect().width : node.offsetWidth;
+      } else {
+        return list.grid ? node.getBoundingClientRect().height : node.offsetHeight;
+      }
+    },
+
+    _itemSizeChanged: function(itemSize) {
+      var dataHost = this._getRootDataHost();
+      this._itemSizeFn = itemSize && (typeof itemSize == 'function' ? itemSize :
+        function() { return dataHost[itemSize].apply(dataHost, arguments); });
```

Comment:
yep, it's a function. Users can bind to it in the same way they can bind to `dom-repeat`'s `sort` or `filter` property.